src/interfaces/records/handlers/records-query.ts

```diff
@@ -152,12 +156,12 @@ export class RecordsQueryHandler implements MethodHandler {
  * @param dateSort - Sorting scheme
  * @returns Sorted Messages
  */
-async function sortRecords(
-  entries: BaseMessage[],
+async function sortRecords<T extends RecordsWriteMessage>(
```

Comment:
I don't necessarily have an argument against this, but it feels maybe overkill since at the moment we only use `sortRecords` in one place.

Reply:
Good call.